cli/util/util_test.go

```diff
@@ -169,11 +170,21 @@ func TestAwaitUtilCancelTx(t *testing.T) {
 
 	e.In.WriteString("one\r")
 	e.Run(t, append(args, txHash.StringLE())...)
-	e.CheckNextLine(t, "Conflicting transaction accepted")
-	resHash, _ := e.CheckAwaitableTxPersisted(t)
 
-	require.Eventually(t, func() bool {
-		_, aerErr := e.Chain.GetAppExecResults(resHash.Hash(), trigger.Application)
-		return aerErr == nil
-	}, time.Second*2, time.Millisecond*50)
+	response := e.GetNextLine(t)
+	if strings.Contains(response, "Conflicting transaction accepted") {
+		resHash, _ := e.CheckAwaitableTxPersisted(t)
+		require.Eventually(t, func() bool {
+			_, aerErr := e.Chain.GetAppExecResults(resHash.Hash(), trigger.Application)
+			return aerErr == nil
+		}, time.Second*2, time.Millisecond*50)
+	} else if strings.Contains(response, "Target transaction accepted") {
+		require.Eventually(t, func() bool {
+			_, _, err := e.Chain.GetTransaction(txHash)
+			require.NoError(t, err, "original transaction should be on chain")
+			return true
+		}, time.Second*2, time.Millisecond*50)
```

Comment:
Should query AER (the same request as for conflicting transaction) or at least check the height is equal to MaxUint32, because `e.Chain.GetTransaction` also searches in mempool.

Let's query AER.